bin/pynumenc

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+"""Convert an input value to/from bytes/numbers."""
+
+import argparse
+import sys
+
+import numenc
+import pynumenc_meta
+
+
+def main() -> None:
```

Comment:
Since this is a command-line utility, please make it return an error code (0 if everything's correct):

```python
def main() -> int:
    ...
```